runtime/filetype.vim

```diff
@@ -1726,7 +1726,7 @@ au BufNewFile,BufRead *.me
 	\   setf nroff |
 	\ endif
 au BufNewFile,BufRead *.tr,*.nr,*.roff,*.tmac,*.mom	setf nroff
-au BufNewFile,BufRead *.[1-9]			call dist#ft#FTnroff()
+au BufRead *.[0-9],*.[013]p,*.[1-8]x,*.3{am,perl,pm,posix,type},*.[nlop]	call dist#ft#FTnroff()
```

Comment:
Why remove `BufNewFile`? People do write new man pages.

Reply:
`dist#ft#FTnroff` checks whether first five lines of the file look like nroff macros, so calling the function is meaningless for a new file.

Reply:
Is it really worth doing such micro-optimizations?